lib/app/features/about/view/about_view.dart

```diff
@@ -30,28 +32,34 @@ https://gordonferguson.org/wp-content/uploads/2016/11/Final-Main-Header.jpg''';
     return ErrorSnackbarView<void>(
       provider: launchProvider(url: _url.value),
       child: Scaffold(
-        appBar: AppBar(title: const Text('About'), actions: [SettingsIconButton()]),
+        appBar: AppBar(
+            title: const Text('About'),
+            centerTitle: Platform.isAndroid,
```

Comment:
We almost certainly should not do this.
android users expect the default left alignment - it's ok that it doesn't match - the only reason you should do something like that is basically if UX asks for it, and dev gives gentle pushback and they say "we really really want it" (oversimplification, but you get the idea)